app/forms/merits_assessments/client_received_legal_help_form.rb

```diff
@@ -6,14 +6,19 @@ class ClientReceivedLegalHelpForm
 
     attr_accessor :client_received_legal_help, :application_purpose
 
-    before_validation :clear_application_purpose
-    validates :client_received_legal_help, presence: true
-    validates :application_purpose, presence: true, if: proc { |form| form.client_received_legal_help.to_s == 'false' }
+    validates :client_received_legal_help, presence: true, unless: :draft?
+    validates(
+      :application_purpose,
+      presence: true,
+      if: proc { |form| !form.draft? && form.client_received_legal_help.to_s == 'false' }
+    )
 
     private
 
     def clear_application_purpose
       application_purpose.clear if client_received_legal_help.to_s == 'true'
     end
+
+    before_validation :clear_application_purpose
```

Comment:
any reason why you moved this all the way down there? I usually see these closer from the top

Reply:
I have no idea. I'll move it back to the top.